README.md

```diff
@@ -1 +1,58 @@
-# StardewValleyPocketTracker
+# SV Go
+
+## Purpose
+This is an android application that is being created to gain experience developing android
+applications written in Java, using git, solidifying programming fundamentals, and designing
+an application from markup to deployment.
+
+## Description
+This is a fan-made project.
+Stardew Valley Pocket tracker keeps track of calender events in Stardew Valley.
+The app also keeps track of items that the user has acquired.
+
+## Screenshots
+These mockups will be replaced with screenshots as features are implemented. All mocks are subject to change as necessary.
+### Home Screen
+The home screen is the default landing page. It will have the app logo, buttons leading to the app's two primary features (Event Viewer and Checklist). The settings button is universally accessible.
+
+![Main Activity](/app/src/main/res/ui-mocks/Main\ Activity.png "Main Activity")
+
+### Settings Screen
+The settings screen is accessible from all activities. When returning from it, the previously viewed activity is displayed.  
+Some settings may be set based on the user activity, such as their event viewer preference. All settings are applied immediately and persist between sessions.  
+If the user chooses to clear all preferences or reset data, they will be asked to confirm.
+
+![Settings Activity](/app/src/main/ui-mocks/Settings.png "Settings Activity")
+
+### Events Screen
+The events screen displays all calendar events in a calendar or list view (pictured below). The user's view preference and last visited month (or first expanded, in list view) is stored as a preference and retrieved when the window is revisited.
+
+![Events Screen Calendar](/app/src/main/ui-mocks/Events\ (Calendar\ View).png "Events Activity - Calendar View") 
+![Events Screen List](/app/src/main/ui-mocks/Events\ (List\ View).png "Events Activity - List View")
```

Comment:
Not a huge fan of the UI design for the event list view. Looks too busy.

Reply:
Agreed - let's revise and review as the feature matures.